pcdsdevices/epics_motor.py

```diff
@@ -874,8 +875,15 @@ def diff_configuration(self, cfgname=None):
         Raises an exception if the comparison fails for any reason.
         """
         IMS._setup_and_check_pmgr()
-        print("Actual Values, Configuration Values")
-        return self._pm.diff_config(self.prefix, cfgname)
+
+        d = self._pm.diff_config(self.prefix, cfgname)
+        table = PrettyTable()
+        table.field_names = ["Parameter", "Actual", "Configuration"]
+        for key, value in d.items():
+            actual = d[key][0]
```

Comment:
`dict.items()` gives you the `key` and `value` as you've set out in the for loop, so you can actually just access the value specifically via: `value[0]` or `value[1]`.  Here you're just assigning `value` and tossing it to the side

Reply:
I made changes to the first point made here but I guess I forgot to push it. This is what the change looks like. 
<img width="1355" alt="Screen Shot 2023-05-01 at 12 38 43" src="https://user-images.githubusercontent.com/35245246/235517613-8f51f7cd-89e2-4db7-a1b5-5e38e56f48d2.png">

Reply:
Looks good 👍   One last suggestion is that you also put `self._id` in the first logger.warning method so we know exactly which object failed.  (also it's technically more readable / "pythonic" to use `is` to check if a value is None)

``` python
if self._stageidentity is not None:
    logger.warning(f"Stage Identity has not been set for object: {self._id}.  Configure manually.")
```

Reply:
Wouldn't it be "is None"?